system-tests/docker-compose.yml

```diff
@@ -171,6 +171,7 @@ services:
       context: ./resources/cli-tools
     volumes:
       - ./resources:/resources
+    tty: true
```

Comment:
Not sure to understand why that is needed

Reply:
That keeps the container running even the entrypoint has finished. It is useful to have a container to connect to for development and testing.